client/src/styles/chat.css

```diff
@@ -66,6 +66,16 @@
         font-weight: revert !important;
     }
 
+    .message-input-container {
+        max-height: 150px; 
+      }
+      
+      #message-input {
+        min-height: 40px; 
+        max-height: 100px; 
+        overflow-y: scroll;
+      }
+
```

Comment:
use tailwind for this

Reply:
It seemed not to work using tailwind but I would try again

Reply:
Sorted out
![Chat jsx - Whisper - Visual Studio Code 10_9_2023 10_34_48 AM](https://github.com/Dun-sin/Whisper/assets/122406759/702b0544-d7f0-41fc-b471-74c820ccd7ab)